src/common/lib/pset.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  AssetHash,
+  ElementsValue,
+  Pset,
+  address as liquidaddress,
+  networks,
+  script,
+} from "liquidjs-lib";
+import { EsploraAPI, EsploraAssetInfos } from "~/common/lib/esplora";
+
+export type Address = {
+  amount: number;
+  address: string;
+  asset: string;
+};
+
+export type PsetPreview = {
+  inputs: Address[];
+  outputs: Address[];
+};
+
+export function getPsetPreview(
```

Comment:
@im-adithya  After our review of the mnemonic feature we moved a lot of code to the background script and tried to avoid introducing new lib files where possible. Is it possible to move `getPsetPreview` to `background-script/liquid` and add an action for it?

Reply:
I moved the pset preview code to the background script in https://github.com/getAlby/lightning-browser-extension/pull/2583

I am not sure about esplora